src/common/cpp/helpers.h

```diff
@@ -8,24 +8,18 @@
 namespace nebula {
 namespace cpp {
 
-class NonCopyable {
- protected:
-  NonCopyable() {}
-  NonCopyable(const NonCopyable&) = delete;
-  NonCopyable& operator=(const NonCopyable&) = delete;
-};
+class NonMovable {
```

Comment:
Is there no `nonmovable` implementation in boost ?

Reply:
yes I search boost and folly codebase ,find nonthing.